orchagent/portsorch.cpp

```diff
@@ -994,6 +994,36 @@ void PortsOrch::getCpuPort(Port &port)
     port = m_cpuPort;
 }
 
+bool PortsOrch::initHostTxReadyState(Port &port)
```

Comment:
This is only updating the state DB table and returning true. Please change it to `void`